Source/WebCore/css/CSSPropertySourceData.h

```diff
@@ -105,14 +104,15 @@ struct CSSRuleSourceData : public RefCounted<CSSRuleSourceData> {
     // Range of the rule body (e.g. style text for style rules) in the enclosing source.
     SourceRange ruleBodyRange;
 
-    // Only for CSSStyleRules.
+    // Only for CSSStyleRules. Not applicable if `isImplicitlyNested`.
     SelectorRangeList selectorRanges;
 
-    // Only for CSSStyleRules, CSSFontFaceRules, and CSSPageRules.
     RefPtr<CSSStyleSourceData> styleSourceData;
 
-    // Only for CSSMediaRules.
     RuleSourceDataList childRules;
+
+    bool isImplicitlyNested = false;
```

Comment:
We generally use brace-initialization in WebCore  `bool isImplicitlyNested { false }`